source/Panel.cpp

```diff
@@ -257,7 +257,7 @@ bool Panel::DoHelp(const string &name) const
 		return false;
 
 	Preferences::Set(preference);
-	ui->Push(new Dialog(message));
+	ui->Push(new Dialog(preference + "\n\n" + message));
```

Comment:
If we're going to display titles, then they shouldn't be prefixed with `help: `, and should also be capitalized, so just giving the preference name isn't sufficient.